_includes/v22.1/backups/retry-failure.md

```diff
@@ -0,0 +1 @@
+If a backup job encounters too many retryable errors, it will enter a [`failed` state](show-jobs.html#job-status) to allow subsequent backups the chance to succeed. When this happens the backup job will fail with the most recent error.
```

Comment:
@mdlinville  Ah I understand what you mean, no there isn't a difference. So, it enters a failed state with the most recent error.

What about something like:

> If a backup job encounters too many retryable errors, it will enter a [`failed` state](show-jobs.html#job-status) with the most recent error, which allows subsequent backups the chance to succeed.

Reply:
Good point, though I think there is a case where you have a backup schedule that is taking backups and you have set the `execution_failure` option to just continue even if there is a failure. 

So, the previous behavior, pausing a backup that is retrying wouldn't necessarily block subsequent backups either. 
The problem with pausing is that you need to resume or cancel the job (manual intervention is required).

 All being said, the definitive "are not blocked from running" might not quite work here, does that make sense? Are you OK with "allowing subsequent backups the chance to succeed"? I might also link to that option I mentioned.

Reply:
SGTM, thanks for your patience while I come up to speed on this type of content. :)

Reply:
Appreciate you bringing stuff up I haven't thought of!